cypress/integration/researchGroupCanSignUp.spec.js

```diff
@@ -0,0 +1,52 @@
+describe("Research Group Sign-Up", () => {
+  beforeEach(function() {
+    cy.server();
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v0/articles",
+      response: "fixture:articles.json",
+      status: 200
+    });
+  });
+  it("User can successfully sign-up as a Research Group w/ Keys", () => {
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/v0/auth",
+      response: "fixture:successful_research_group_sign_up.json"
+    });
+    cy.research_group_successful_signup(
+      "research_group",
+      "Cancer Research Group",
+      "cancer_research@mail.com",
+      "password",
+      "password",
+      "abcd1234"
+    );
+    cy.get("#submit-account-button").click();
+    cy.get("#flash").should(
+      "contain",
+      "Your research_group account successfully created!"
+    );
+  });
+  it("User can't sign up without valid Subscription Key", () => {
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/v0/auth",
+      response:"fixture:unsuccessful_research_group_sign_up.json",
+      status: 422
```

Comment:
I believe it should be a 200 response actually. Since its a ”successful” response when the RG cant sign up with the wrong key

Reply:
Ah, I see what you mean.  I need to rename the test. . .we are definitely expecting a 422 error because the test assumes that the user puts in a bad key, so an error message should come back from the server.  I've fixed the test name so it makes more sense.  Thanks for pointing this out!!